### PR TITLE
feat(notifications): add notifications by types, add toasts notificat…

### PR DESCRIPTION
**Pull request by VERZUOL1**
…ions

### CODE REVIEW

File: types/notification.ts [hunk 1]
```diff
@@ -72,28 +70,38 @@ export interface NotificationDisableOption {
...
+  RemoveMemberFromRole = 'RemoveMemberFromRole',
+  FunctionCall = 'FunctionCall',
+  Transfer = 'Transfer',
+  ChangePolicy = 'ChangePolicy',
+  ChangeConfig = 'ChangeConfig',
+  AddBounty = 'AddBounty',
+  BountyDone = 'BountyDone',
+  Vote = 'Vote',
+
+  // TransferProposalCreation = 'TransferProposalCreation',
```
Comment (AlexeyPonomarenko) [inline types/notification.ts:89]: Should we keep this comments?

File: features/proposal/components/ProposalActions/ProposalActions.module.scss [hunk 2]
```diff
@@ -52,6 +52,10 @@
   }
 }
 
+.inactive button {
```
Comment (AlexeyPonomarenko) [inline features/proposal/components/ProposalActions/ProposalActions.module.scss]: I consider we should avoid tags in css.

File: astro_2.0/features/Notifications/helpers.tsx [hunk 3]
```diff
@@ -369,3 +363,85 @@ export function mapDelayToTime(delay: string): number {
...
+      const actioner =
+        accountId === proposerId ? 'Your' : extractPrefix(proposerId);
+
+      return `${actioner}'s ${type} ${t('proposalWasRejected', {
+        dao: dao.displayName || extractPrefix(dao.id),
+      })}`;
+    }
+    case NotificationStatus.Approved: {
+      const actioner =
+        accountId === proposerId ? 'Your' : extractPrefix(proposerId);
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/Notifications/helpers.tsx]: I see that some strings are localised and some are not. Not sure if you've left some intentionally.

File: astro_2.0/features/Notifications/helpers.tsx [hunk 4]
```diff
@@ -81,33 +140,33 @@ export function extractPrefix(value: string, delimiter?: string): string {
...
-  NotifiedActionType.DaoLinksUpdated,
-  NotifiedActionType.DaoFlagUpdated,
-  NotifiedActionType.DaoDeadlinesUpdated,
-  NotifiedActionType.DaoRulesUpdated,
-  NotifiedActionType.DaoGroupAdded,
-  NotifiedActionType.DaoGroupUpdated,
-  NotifiedActionType.DaoGroupRemoved,
-  NotifiedActionType.DaoMembersAdded,
-  NotifiedActionType.DaoMemberRemoved,
+// AddMemberToRole = 'AddMemberToRole',
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/Notifications/helpers.tsx]: Should we keep this comments here?

File: astro_2.0/features/Notifications/helpers.tsx [hunk 5]
```diff
@@ -6,51 +6,83 @@ import {
...
+    case NotifiedActionType.Transfer: {
       noteType = NotificationType.Transfer;
       url = `/dao/${daoId}/proposals/${targetId}`;
 
       break;
     }
-    case NotifiedActionType.PollProposalCreation: {
+    case NotifiedActionType.FunctionCall: {
+      noteType = NotificationType.Transfer;
+      url = `/dao/${daoId}/proposals/${targetId}`;
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/Notifications/helpers.tsx]: I consider we should use route constants from`routing.ts` to keep them consistent.

**Review by AlexeyPonomarenko — DISMISSED** [review]

File: types/notification.ts (hunk 1 above)
Comment (VERZUOL1) [reply types/notification.ts:89]: These are the old types, I kept them just for reference. Ill remove them once we agreed about all possible notifications

File: astro_2.0/features/Notifications/helpers.tsx (hunk 5 above)
Comment (VERZUOL1) [reply astro_2.0/features/Notifications/helpers.tsx]: fixed

File: astro_2.0/features/Notifications/helpers.tsx (hunk 4 above)
Comment (VERZUOL1) [reply astro_2.0/features/Notifications/helpers.tsx]: fixed

File: astro_2.0/features/Notifications/helpers.tsx (hunk 3 above)
Comment (VERZUOL1) [reply astro_2.0/features/Notifications/helpers.tsx]: fixed

File: features/proposal/components/ProposalActions/ProposalActions.module.scss (hunk 2 above)
Comment (VERZUOL1) [reply features/proposal/components/ProposalActions/ProposalActions.module.scss]: fixed

**Review by AlexeyPonomarenko — APPROVED** [review]

**Pull request merged**